apps/web/src/components/Icon/Icon.stories.tsx

```diff
@@ -1,7 +1,8 @@
-import Icon, { iconColor } from './Icon';
+import Icon, { iconColor, Props } from './Icon';
```

Comment:
If you type this story correctly (`StoryFn`, `Meta`) you shouldn't need to bring in `Props` from `Icon`- the type should be inferred.